plugins/forc-index/src/commands/init.rs

```diff
@@ -6,29 +6,29 @@ use std::path::PathBuf;
 /// Create a new indexer project in the current directory.
 #[derive(Debug, Parser)]
 pub struct Command {
-    /// Name of index
-    #[clap(long, help = "Name of index.")]
+    /// Name of indexer
+    #[clap(long, help = "Name of indexer.")]
     pub name: Option<String>,
 
-    /// Path at which to create index
+    /// Path at which to create indexer
     #[clap(
         short,
         long,
         parse(from_os_str),
-        help = "Path at which to create index."
+        help = "Path at which to create indexer."
     )]
     pub path: Option<PathBuf>,
 
-    /// Namespace in which index belongs.
-    #[clap(long, help = "Namespace in which index belongs.")]
+    /// Namespace in which indexer belongs.
+    #[clap(long, help = "Namespace in which indexer belongs.")]
```

Comment:
In the file below `new.rs` we have `/// Name of the indexer namespace`. I'm neutral on which one we choose, but they should be the same for consistency.